mas/src/androidTest/java/com/ca/mas/MASCallbackFuture.java

```diff
@@ -0,0 +1,76 @@
+/*
```

Comment:
I feel like this should be renamed MASFutureCallback since it still extends from MASCallback. Maybe also add a class comment on what this is trying to achieve or be used for?